core/src/main/java/nl/nn/adapterframework/jms/JMSFacade.java

```diff
@@ -81,38 +81,38 @@
 	private boolean jmsTransacted = false;
 	private String subscriberType = "DURABLE"; // DURABLE or TRANSIENT
 
-	private int ackMode = Session.AUTO_ACKNOWLEDGE;
-	private boolean persistent;
-	private long messageTimeToLive = 0;
-	private String destinationName;
-	private boolean useTopicFunctions = false;
-	private String authAlias;
-	private boolean lookupDestination = true;
-
-	private String destinationType = "QUEUE"; // QUEUE or TOPIC
-
-	protected MessagingSource messagingSource;
-	private Destination destination;
-
-	private Map<String, ConnectionFactory> proxiedConnectionFactories;
-	private Map<String, String> proxiedDestinationNames;
-
-	// ---------------------------------------------------------------------
-	// Queue fields
-	// ---------------------------------------------------------------------
-	private String queueConnectionFactoryName;
-	// ---------------------------------------------------------------------
-	// Topic fields
-	// ---------------------------------------------------------------------
-	private String topicConnectionFactoryName;
-
-	// the MessageSelector will provide filter functionality, as specified
-	// javax.jms.Message.
-	private String messageSelector = null;
-
-	private boolean correlationIdToHex = false;
-	private String correlationIdToHexPrefix = "ID:";
-	private int correlationIdMaxLength = -1;
+    private int ackMode = Session.AUTO_ACKNOWLEDGE;
```

Comment:
Zijn al deze changes puur tabs die omgezet zijn naar spaties?

Reply:
Volgens mij is de merge niet helemaal goed gegaan of staat bij jou 'show whitespaces' uit. In de pr lijkt het nu alsof gigantisch veel veranderd is